test/parallel/test-fs-cp.mjs

```diff
@@ -336,6 +336,24 @@ if (!isWindows) {
   }, { code: 'ERR_INVALID_RETURN_VALUE' });
 }
 
+// It should not throw exception if child folder
+// does not pass filter function
+{
+  // Mimic there's a file in dest with the same name as the child folder in src
```

Comment:
The start of this sentence is a bit awkward.

Reply:
Updated